project.clj

```diff
@@ -1,23 +1,23 @@
 ;; Please don't bump the library version by hand - use ci.release-workflow instead.
-(defproject com.nedap.staffing-solutions/components.assets "3.0.1-alpha1"
+(defproject com.nedap.staffing-solutions/components.assets "3.0.1"
   ;; Please keep the dependencies sorted a-z.
-  :dependencies [[com.nedap.staffing-solutions/stefon "0.5.2"
-                  :exclusions [clj-time commons-codec com.fasterxml.jackson.core/jackson-core]]
-                 [com.nedap.staffing-solutions/utils.io "2.0.0"
-                  :exclusions [com.nedap.staffing-solutions/speced.def
-                               org.apache.commons/commons-compress]]
+  :dependencies [[circleci/stefon "0.5.0-SNAPSHOT"]
```

Comment:
Lets not depend on a SNAPSHOT.

Also we should make sure to mention the downgrade of Stefon in the release, and maybe mention using a git dependency in the readme

Reply:
Other's wouldn't work iirc

Reply:
https://clojars.org/circleci/stefon/versions/ allows us to pin a SNAPSHOT. Still subpar, but for a repo that wasn't touched in years, good enough